docs/module/isis.md

```diff
@@ -30,13 +30,15 @@ The following table describes per-platform support of individual IS-IS features:
 | Cisco IOS          | ✅  | ✅  | ✅  | ✅  | ✅  | ✅ | 
 | Cisco IOS XE       | ✅  | ✅  | ✅  | ✅  | ✅  | ✅ | 
 | Cisco Nexus OS     | ✅  | ✅  | ✅  | ✅  | ✅  | ❌  |
+| Cisco ASAv         | ✅  | ✅  | ✅  | ❌  | ❌  | ❌ |
 | Nokia SR Linux     | ✅  | ✅  | ✅  | ✅  | ✅  | ✅ | 
 | Nokia SR OS        | ✅  | ✅  | ✅  | ✅  | ✅  | ✅ | 
 | VyOS               | ✅  | ✅  | ✅  | ❌  | ✅  | ✅ |
 
 **Notes:**
 * On Arista EOS, IPv6 is enabled on all interfaces as soon as one interface has an IPv6 address. Arista EOS implementation of IS-IS refuses to work on interfaces with missing address families.
 * On VyOS, IPv6 is enabled on all interfaces as soon as one interface has an IPv6 address.
+* On Cisco ASA you have to add `isis.network_type: false` on all point-to-point links to devices with other NOS, otherwise the ASA will not recognize the IS-IS packets
```

Comment:
This looks like something to be solved within the ASA template logic. Can't ask users to change their design to accommodate quirks on some device

Reply:
The problem is that I need a different configuration on the remote side, because the ASA has only implemented broadcast IS-IS packets.
So if I want to connect an ASA with an ios or an nexus I must remove the command `isis network point-to-point` within the ios/nxos config.

Reply:
That's what we have the 'quirks' package for, see e.g. https://github.com/ipspace/netlab/blob/dev/netsim/quirks/eos.py#L19

What we need to add, is some logic for ASA that throws an error when the user is trying to use anything besides 'broadcast' for IS-IS

In general this would be modeled as a feature flag, but since this platform is the only one that doesn't support all types a 'quirk' is the better solution

Reply:
@jbemmel: In principle, I agree with you, and it would be great to have all caveats covered in quirks. In practice, I appreciate the great work @willimei did ;)

Also, I reworded the documentation a bit to explain what the caveat is.

Reply:
Sure, I did not mean to imply a lack of appreciation - just nit-picking

Reply:
I was not aware of the quirks package until now. Thanks for that hint.
I will have a look into it tomorrow.